packages/web/src/scss/components/Collapse/_Collapse.scss

```diff
@@ -23,6 +23,11 @@
         overflow: visible;
     }
 
+    /* When it should be as a hidden inline element */
```

Comment:
@adamkudrna @crishpeen I'm not entirely sure about this one. The point is that the example for the implementation with `data-more` was a `div` element inside a `p`. But even though `span` is set to `display: block`, that's why I set it to be hidden when the element is `span`.

Reply:
I'm not sure if I understand everything correctly but I'll try to explain myself as clearly as I can 🙂.

> The point is that the example for the implementation with data-more was a div element inside a p.

That doesn't sound good! Where do we have `div` inside a `p`? I cannot find it…

> This is also related to the internal element, which should be the same as the parent. But in the twig and react components we have the option to set only the parent element. So I suggest that the same type of element should be applied to the inner element as well.

Nice catch! 👍🏻  You're right, we don't want to allow `div`s inside `span`s as such HTML is invalid.

Reply:
@adamkudrna Based on that, I created a [new task](https://jira.lmc.cz/browse/DS-541) to supplement that property. For twig, I can still finish it while I have it divided.